mappings/net/minecraft/screen/ForagingScreenSlots.mapping

```diff
@@ -0,0 +1,35 @@
+CLASS net/minecraft/class_8047 net/minecraft/screen/ForagingScreenSlots
+	FIELD field_41907 inputSlots Ljava/util/List;
+	FIELD field_41908 resultSlot Lnet/minecraft/class_8047$class_8049;
+	METHOD <init> (Ljava/util/List;Lnet/minecraft/class_8047$class_8049;)V
+		ARG 1 inputSlots
+		ARG 2 resultSlot
+	METHOD method_48364 builder ()Lnet/minecraft/class_8047$class_8048;
+	METHOD method_48365 has (I)Z
+		ARG 1 index
+	METHOD method_48366 getResult ()Lnet/minecraft/class_8047$class_8049;
+	METHOD method_48367 get (I)Lnet/minecraft/class_8047$class_8049;
+		ARG 1 index
+	METHOD method_48368 getInputs ()Ljava/util/List;
+	METHOD method_48369 getInputSlotCount ()I
+	METHOD method_48370 getInputSlotCount2 ()I
```

Comment:
What's the difference between these two methods? Usually, methods don't have numeric suffixes.